roles/storage/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# defaults file for storage
+# Mount point for home directories
+home_location: /home
+# Mount point for alternative home directories, used for syncing old home and as bind mount source
+alt_home_location: /data/volume_3
```

Comment:
This comment is a bit unclear to me. Can we explain better what the relation between home and alt_home is?

Reply:
Thanks, this is great!